test/remote/gateways/remote_cecabank_rest_json_test.rb

```diff
@@ -150,23 +151,28 @@ def test_failure_stored_credential_invalid_cit_transaction_id
 
   def three_d_secure
     {
-      authentication_value: '4F80DF50ADB0F9502B91618E9B704790EABA35FDFC972DDDD0BF498C6A75E492',
+      version: '2.2.0',
+      eci: '02',
+      cavv: '4F80DF50ADB0F9502B91618E9B704790EABA35FDFC972DDDD0BF498C6A75E492',
+      ds_transaction_id: 'a2bf089f-cefc-4d2c-850f-9153827fe070',
+      acs_transaction_id: '18c353b0-76e3-4a4c-8033-f14fe9ce39dc',
+      authentication_response_status: 'Y',
+      three_ds_server_trans_id: '9bd9aa9c-3beb-4012-8e52-214cccb25ec5'
+    }
+  end
+
+  def extra_options_for_three_d_secure
```

Comment:
💬 Comment

Two things here:

1) there are some values that are available as part of the request like `email` | `amount` | `ip` | etc.

2) As a consequence of the comment related to available params this value is not going to represent the actual result of the 3DS authentication by Seglan.